test/tape/transactions.js

```diff
@@ -465,20 +449,19 @@ module.exports = function(knex) {
               })
               .catch(function() {});
           });
-      })
-      .finally(function() {
-        t.equal(
-          secondTransactionCompleted,
-          true,
-          'Second sibling transaction completed'
-        );
-        return knex('test_table').then(function(results) {
-          t.equal(results.length, 1, 'Parent transaction inserted one record');
-        });
       });
+    } finally {
+      t.equal(
+        secondTransactionCompleted,
+        true,
+        'Second sibling transaction completed'
+      );
+      const results = await knex('test_table');
+      t.equal(results.length, 1, 'Parent transaction inserted one record');
+    }
   });
 
-  test('#625 - streams/transactions', 'postgresql', function(t) {
+  test('#625 - streams/transactions', 'postgresql', (t) => {
```

Comment:
Just curious: was there a reason that this needed to be changed into an arrow function?

Reply:
In general we are replacing anonymous functions with arrow functions, so there didn't have to be need.